src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/UserEntity.kt

```diff
@@ -154,7 +154,70 @@ interface UserRepository : JpaRepository<UserEntity, UUID>, JpaSpecificationExec
           a.allocated_to_user_id = u.id
           and a.reallocated_at is null
           and a.submitted_at > current_date - interval '30' day
-      ) as completedAssessmentsInTheLastThirtyDays
+      ) as completedAssessmentsInTheLastThirtyDays,
+      (
+        SELECT
+          count(*)
+        from
+          placement_applications placement_application
+        where
+          placement_application.allocated_to_user_id = u.id
+          and placement_application.reallocated_at is null
+          and placement_application.submitted_at is null
+      ) as pendingPlacementApplications,
+      (
+        SELECT
+          count(*)
+        from
+          placement_applications placement_application
+        where
+          placement_application.allocated_to_user_id = u.id
+          and placement_application.reallocated_at is null
+          and placement_application.submitted_at > current_date - interval '7' day
+      ) as completedPlacementApplicationsInTheLastSevenDays,
+      (
+        SELECT
+          count(*)
+        from
+          placement_applications placement_application
+        where
+          placement_application.allocated_to_user_id = u.id
+          and placement_application.reallocated_at is null
+          and placement_application.submitted_at > current_date - interval '30' day
+      ) as completedPlacementApplicationsInTheLastThirtyDays,
+      (
+        SELECT
+          count(*)
+        from
+          placement_requests placement_request
+        where
+          placement_request.allocated_to_user_id = u.id
+          and placement_request.booking_id is null
+          and placement_request.reallocated_at is null
+      ) as pendingPlacementRequests,
+      (
+        SELECT
+          count(*)
+        from
+          placement_requests placement_request
+          left join bookings booking on booking.id = placement_request.booking_id
+        where
+          placement_request.allocated_to_user_id = u.id
+          and placement_request.booking_id is not null
+          and placement_request.reallocated_at is null
+          and booking.created_at > current_date - interval '7' day
+      ) as completedPlacementRequestsInTheLastSevenDays,
+      (
+        SELECT
+          count(*)
+        from
+          placement_requests placement_request
+          left join bookings booking on booking.id = placement_request.booking_id
+        where
+          placement_request.allocated_to_user_id = u.id
+          and placement_request.reallocated_at is null
+          and booking.created_at > current_date - interval '30' day
```

Comment:
the 7 day version of this query had 'placement_request.booking_id is not null', but this one doesn't. Is that deliberate?

Reply:
Good catch!